app/Repositories/Eloquents/BookEloquentRepository.php

```diff
@@ -255,4 +255,22 @@ public function search($attribute, $data, $with = [], $dataSelect = ['*'])
             ->search($attribute, $data)
             ->get();
     }
+
+    public function updateStar($data, $id)
+    {
+        $book = $this->model()->findOrFail($id);
+
+        $star = 0;
+        foreach ($data as $review) {
+            $star += $review->star;
+        }
+
+        if ($data->count() > 0) {
+            $Avgstar['avg_star'] = $star/count($data);
+        } else {
+            $Avgstar['avg_star'] = $star;
+        }
+
+        return $book->update($Avgstar);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookEloquentRepository.php</br>Variable "Avgstar" is not in valid camel caps format